src/App.js

```diff
@@ -12,6 +12,7 @@ function App() {
         <Route path="/" element={<Books />} />
         <Route path="/categories" element={<Categories />} />
       </Routes>
+      {/* <Books /> */}
```

Comment:
- [ ] Just as I pointed out above about Following the best practices, you are not meant to leave unused code as a comment on your code base. I'll suggest you remove this as it makes your codebase look a bit a bit unprofessional. :+1: :smile: